src/serlo-editor/store/store.ts

```diff
@@ -19,7 +19,10 @@ export const store = configureStore({
     getDefaultMiddleware({
       // https://github.com/serlo/backlog/issues/127
       serializableCheck: {
-        ignoredActions: ['documents/runChangeDocumentSaga'],
+        ignoredActions: [
+          'documents/runChangeDocumentSaga',
+          'history/commitTemporaryActionToHistory',
```

Comment:
basically this does not change anything (aside from hiding warnings and indicating to us that we should investigate at some point? :)